packages/codegen-ui/lib/__tests__/generate-form-definition/helpers/position.test.ts

```diff
@@ -59,3 +64,62 @@ describe('removeAndReturnItemOnward', () => {
     expect(formDefinition.elementMatrix).toStrictEqual([['one'], ['five'], ['six', 'seven', 'eight']]);
   });
 });
+
+describe('mapElementMatrix', () => {
+  it('vertical - should throw if element is positioned relative to non-existing name', () => {
```

Comment:
minor style nit, vertical/horizontal can be their own describe groups

Reply:
Makes sense. Will rev in next PR